<launch>
    <!-- Motion Launch Node Controller -->
    <node pkg="tb3_motion" type="motion_core.py" name="motion_core" output="screen" />

</launch>